transmartApp/grails-app/controllers/org/transmartproject/interceptors/ConceptsInterceptor.groovy

```diff
@@ -1,12 +1,11 @@
 package org.transmartproject.interceptors
 
-import org.transmart.audit.AuditLogService
 import org.transmartproject.core.users.User
 
 
 class ConceptsInterceptor {
 
-    AuditLogService auditLogService
+    def auditLogService
```

Comment:
Adding by-type spring bean insertion will force adding dependency on transmart-server, so we will have a circular dependency there (between tm-app and tm-server). Maybe it's better to move this bean definition somewhere else?